### PR TITLE
Review PR for node-docs.md

### PR DESCRIPTION
**Pull request by dominicbetts**
# Guidelines for reviewers
Using this Pull Request (PR), you can:

- Add general comments about the article in this PR
- Add a comment to a specific line in one of the files that make up this article
- Propose edits to the article

You can view the published version of this article at:
https://azure.microsoft.com/documentation/articles/node-docs/

## Add general comments
To add a general comment, use this page (**Conversation** tab)

## Add comments to a specific line
To add a comment to specific line, go to the **Files changed** tab, hover your mouse pointer over the line in the file and click the blue **+** to add a new line comment.

## Propose edits to the article
To propose edits to this article, go to the **Files changed** tab, find the file you want to edit and click the pencil icon (**Change this file using the online editor**), and make your changes. When you are done you will be prompted to create a PR that contains your proposed changes.

> Note: The PR with your proposed changes will be pushing changes to the dominicbetts/azure-content-pr:edit-node-docs branch.

### CODE REVIEW

File: node/device/doc/devbox_setup.md [hunk 1]
```diff
@@ -0,0 +1 @@
+**Note:** The content you are looking for is now located at: <https://github.com/Azure/azure-iot-sdks/blob/master/doc/get_started/node-devbox-setup.md>.
```
Comment (nberdy) [inline node/device/doc/devbox_setup.md]: Can we just fix the original links so we no longer need these notices?

File: node/device/doc/run_sample.md [hunk 2]
```diff
@@ -0,0 +1 @@
+**Note:** The content you are looking for is now located at: <https://github.com/Azure/azure-iot-sdks/blob/master/doc/get_started/node-run-sample.md>.
```
Comment (nberdy) [inline node/device/doc/run_sample.md]: Can we just fix the original links so we no longer need these notices?

File: node/device/doc/run_sample_on_toradex_linux.md [hunk 3]
```diff
@@ -0,0 +1 @@
+**Note:** The content you are looking for is now located at: <https://github.com/Azure/azure-iot-sdks/blob/master/doc/get_started/node-run-sample-toradex-linux.md>.
```
Comment (nberdy) [inline node/device/doc/run_sample_on_toradex_linux.md]: Can we just fix the original links so we no longer need these notices?

**Comment by dominicbetts** [comment]
I've created a work item to do this throughout the whole repo.

**Comment by dominicbetts** [comment]
Processed all feedback up to here.

**Pull request closed without merging**